Description: Search for HDF5 library in other paths too.
 The HDF5 libraries are installation in non-standard locations in Debian,
 not in the same directory as the NetCDF library.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: http://gmt.soest.hawaii.edu/issues/732

--- a/cmake/modules/FindNETCDF.cmake
+++ b/cmake/modules/FindNETCDF.cmake
@@ -105,11 +105,21 @@ find_library (NETCDF_LIBRARY
 	/opt
 )
 
+if (NOT DEFINED HDF5_LIBRARY_DIRS)
+  set (HDF5_LIBRARY_DIRS "")
+endif (NOT DEFINED HDF5_LIBRARY_DIRS)
+
+if (NOT DEFINED HDF5_INCLUDE_DIRS)
+  set (HDF5_INCLUDE_DIRS "")
+else (NOT DEFINED HDF5_INCLUDE_DIRS)
+  list (APPEND NETCDF_INCLUDE_DIR ${HDF5_INCLUDE_DIRS})
+endif (NOT DEFINED HDF5_INCLUDE_DIRS)
+
 # find all libs that nc-config reports
 foreach (_extralib ${_netcdf_lib})
 	find_library (_found_lib_${_extralib}
 		NAMES ${_extralib}
-		PATHS ${_netcdf_libpath})
+		PATHS ${_netcdf_libpath} ${HDF5_LIBRARY_DIRS})
 	list (APPEND NETCDF_LIBRARY ${_found_lib_${_extralib}})
 endforeach (_extralib)
 
